packages/control-property-editor/jest.config.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
```

Comment:
Please extend the base condfig (`const config = require('../../jest.base'); config. ...`)

Reply:
Maybe have a look at https://github.com/SAP/open-ux-tools/blob/main/packages/ui-components/jest.config.js, I would assume you will required almost the same.

Reply:
Adapted.